sodetlib/operations/translate_tes_parameters.py

```diff
@@ -0,0 +1,569 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.optimize import curve_fit
+from sodetlib.operations.iv import IVAnalysis
+from sodetlib.operations.bias_steps import BiasStepAnalysis
+from tqdm.auto import trange
+
+
+import warnings
+warnings.filterwarnings('ignore')
+np.seterr(all="ignore")
+
+
+def translate_tes_params(iv_file, bs_file, save_iv=False, save_bs=False):
+    """
+    Driver function that recalculates IV TES parameters using the
+    thevenin equivalent voltage. Then computes the RP curve following
+    Satoru's methods. Finally, load in and recalculate bias step analysis
+    TES parameters using a shifted RP curve determined by the change in 
+    optical power between operations. 
+    
+    Args:
+    -----
+    iv_file : str
+        filepath to iv_analysis.npy file to re-analyze.
+        
+    bs_file : str
+        filepath to bs_analysis.npy file to re-analyze
+    
+    save_iv : bool
+        Boolean that determines if the re-analyzed iva is saved
+        
+    save_bs : bool
+        Boolean that determines if the re-analyzed bsa is saved
+    """
+    iva2 = reanalyze_iv(iv_file, save_iv)
+    bsa2 = rerun_analysis(bs_file, iva2, save_bs)
```

Comment:
To both the iva and bsa objects, can you add a new key `version` which should be set to 1? Then this can be checked to make sure corrections aren't re-applied to already corrected results.